src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574663,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040004
+name: PIK3R2-related overgrowth spectrum
+def: "Any overgrowth syndrome resulting from pathogenic gain-of-function variants in the PIK3R2 gene. The variants can be germline or somatic" [https://clinicalgenome.org/affiliation/50020/, https://orcid.org/0000-0002-0587-4693, PMID:22729224, PMID:26860062, PMID:27854409, PMID:32856318]
+synonym: "overgrowth syndrome and/or cerebral malformations due to abnormalities in MTOR pathway genes" RELATED [https://clinicalgenome.org/affiliation/50020/, PMID:22729224, PMID:26860062, PMID:27854409]
+is_a: MONDO:0019716 {source="PMID:22729224", source="PMID:26860062", source="PMID:27854409", source="PMID:32856318", source="https://clinicalgenome.org/affiliation/50020/"} ! overgrowth syndrome
+property_value: http://purl.org/dc/elements/1.1/date "2024-07-03T14:16:10Z" xsd:dateTime
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+
```

Comment:
please add the term tracker item to link to the ticket:
https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#term-tracker-item

Reply:
Added term tracker! Thanks for the reminder